### ISSUE TITLE
Bug: in firefox, Mathjax menu labels not announced correctly via screen reader

### ISSUE THREAD

**Issue by RichCaloggero** [mathjax/MathJax#3400]
### Issue Summary

In latest firefox, arrowing through the menu with NVDA often cuts off the announcement of the menu label. DOes not happen with all menu items, and sometimes is dependent on when you are arrowing up / down or exactly which menu item gains focus.

### Steps to Reproduce

1. open any page using MathJax 3.2.2 with NVDA running in firefox
   - try https://richcaloggero.github.io/test/mathJaxTest.html
2. tab to an expression and trigger the menu
3. arrow around in the menu and submenus
   - accessibility is often announced as just "a"
   - show math menu is often cut off
   - items within the accessibility menu are often truncated
   - "about" and "help" menus are always announced correctly

This is a bug because menu items are not read properly. This behavior not seen in Chrome!

### Technical details

* MathJax Version: 3.2.2
* Client OS: windows 10
* Browser: Firefox
